app/javascript/utilities/videoPlayback.js

```diff
@@ -12,12 +12,14 @@
 /* eslint no-use-before-define: 0 */
 /* eslint no-param-reassign: 0 */
 /* eslint no-useless-escape: 0 */
-/* global jwplayer, ahoy */
+/* global jwplayer */
 
-function initializeVideoPlayback() {
-  var currentTime = '0';
-  var deviceType = 'web';
-  var lastEvent = '';
+import ahoy from 'ahoy.js';
+
+export function initializeVideoPlayback() {
```

Comment:
Same as above: Now that this was moved to webpack bundle there's a bunch of small changes to ensure eslint passes (i.e. var -> let/const)